### PR TITLE
LPS-126646 | Master

### PR DESCRIPTION
**Pull request by nkkwon**
https://issues.liferay.com/browse/LPS-126646
@david-gutierrez-mesa 
@liulukiwi
@Tim-Cao

### CODE REVIEW

**Comment by nkkwon** [comment]
ci:test:sf

**Comment by nkkwon** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/macros/journal/template/WebContentStructures.macro [hunk 1]
```diff
@@ -111,6 +111,14 @@ definition {
 		}
 	}
 
+	macro editStructure {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/journal/template/WebContentStructures.macro:114]: Duplicate with WebContentNavigator.gotoEditStructure

**Comment by Tim-Cao** [comment]
@david-gutierrez-mesa, Could you tell more detailed about the steps? Just edit structure name or edit field label and other properties?

**Comment by david-gutierrez-mesa** [comment]
Hi @Tim-Cao and @nkkwon The idea is to check structure files. They are the more affected by Date Engine Migration. Sorry for not detail it better in the test case definition.

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3565

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**